Maps.php

```diff
@@ -109,40 +127,19 @@
 	// Parser hooks
 
 	// Required for #coordinates.
-	$GLOBALS['wgHooks']['ParserFirstCallInit'][] = function( Parser &$parser ) {
-		$instance = new MapsCoordinates();
-		return $instance->init( $parser );
-	};
-
-	$GLOBALS['wgHooks']['ParserFirstCallInit'][] = function( Parser &$parser ) {
-		$instance = new MapsDisplayMap();
-		return $instance->init( $parser );
-	};
-
-	$GLOBALS['wgHooks']['ParserFirstCallInit'][] = function( Parser &$parser ) {
-		$instance = new MapsDistance();
-		return $instance->init( $parser );
-	};
-
-	$GLOBALS['wgHooks']['ParserFirstCallInit'][] = function( Parser &$parser ) {
-		$instance = new MapsFinddestination();
-		return $instance->init( $parser );
-	};
-
-	$GLOBALS['wgHooks']['ParserFirstCallInit'][] = function( Parser &$parser ) {
-		$instance = new MapsGeocode();
-		return $instance->init( $parser );
-	};
-
-	$GLOBALS['wgHooks']['ParserFirstCallInit'][] = function( Parser &$parser ) {
-		$instance = new MapsGeodistance();
-		return $instance->init( $parser );
-	};
-
-	$GLOBALS['wgHooks']['ParserFirstCallInit'][] = function( Parser &$parser ) {
-		$instance = new MapsMapsDoc();
-		return $instance->init( $parser );
-	};
+	$GLOBALS['wgHooks']['ParserFirstCallInit'][] = 'MapsHooks::onParserFirstCallInit1';
+
+	$GLOBALS['wgHooks']['ParserFirstCallInit'][] = 'MapsHooks::onParserFirstCallInit2';
+
+	$GLOBALS['wgHooks']['ParserFirstCallInit'][] = 'MapsHooks::onParserFirstCallInit3';
+
+	$GLOBALS['wgHooks']['ParserFirstCallInit'][] = 'MapsHooks::onParserFirstCallInit4';
+
+	$GLOBALS['wgHooks']['ParserFirstCallInit'][] = 'MapsHooks::onParserFirstCallInit5';
+
+	$GLOBALS['wgHooks']['ParserFirstCallInit'][] = 'MapsHooks::onParserFirstCallInit6';
+
+	$GLOBALS['wgHooks']['ParserFirstCallInit'][] = 'MapsHooks::onParserFirstCallInit7';
```

Comment:
Needs to be done otherwise we would have duplicate code and would need to update it in two places.